dialogue-core/src/main/java/com/palantir/dialogue/core/ConcurrencyLimitedChannel.java

```diff
@@ -39,16 +42,19 @@
 final class ConcurrencyLimitedChannel implements LimitedChannel {
     private static final SafeLogger log = SafeLoggerFactory.get(ConcurrencyLimitedChannel.class);
 
+    private static final StateHolderKey<ConcurrencyLimitedChannelState> STATE_HOLDER_KEY =
+            new StateHolderKey<>(ConcurrencyLimitedChannelState.class, ConcurrencyLimitedChannel::createState);
```

Comment:
II think we should include `HOST` in the name, since this is creating a host-specific limiter

Reply:
👍 done